<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Don</class>
 <widget class="QDialog" name="Don">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Don</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>901</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Dons</string>
    </attribute>
    <widget class="QComboBox" name="ID_DONATEUR">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MONTANT_DON">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>230</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>ID Donateur</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableView" name="data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>401</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>ID Don</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Par:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer_3">
     <property name="geometry">
      <rect>
       <x>614</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>ID_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MONTANT_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID_DONATEUR</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_DON">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(233,248,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri√© Par:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNoPredictiveText</set>
     </property>
     <property name="placeholderText">
      <string>Recherche...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer_4">
     <property name="geometry">
      <rect>
       <x>724</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DATE">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>260</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ID_DONATEUR_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MONTANT_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID_DONATEUR</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Montant</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>ID_DONATEUR</zorder>
    <zorder>MONTANT_DON</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>data</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>pb_supprimer_3</zorder>
    <zorder>comboBox</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>ID_DON</zorder>
    <zorder>lineEdit_id_6</zorder>
    <zorder>pb_supprimer_4</zorder>
    <zorder>DATE</zorder>
    <zorder>ID_DONATEUR_2</zorder>
    <zorder>label_4</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Donateurs</string>
    </attribute>
    <widget class="QComboBox" name="REF">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PRENOM">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Montant</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>230</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Ref_Don</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableView" name="data_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>401</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Par:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer_5">
     <property name="geometry">
      <rect>
       <x>614</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REF_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MONTANT</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_ajouter_2">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NOM">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(233,248,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri√© Par:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNoPredictiveText</set>
     </property>
     <property name="placeholderText">
      <string>Recherche...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer_6">
     <property name="geometry">
      <rect>
       <x>724</x>
       <y>370</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ID_DONATEUR_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REF_DON</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MONTANT</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>ID Donateur</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_DONATEUR_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MONTANT">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>REF</zorder>
    <zorder>PRENOM</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>data_2</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>pb_supprimer_5</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>pb_ajouter_2</zorder>
    <zorder>NOM</zorder>
    <zorder>lineEdit_id_7</zorder>
    <zorder>pb_supprimer_6</zorder>
    <zorder>ID_DONATEUR_4</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>ID_DONATEUR_3</zorder>
    <zorder>MONTANT</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
